docs/user/field-selectors.md

```diff
@@ -0,0 +1,149 @@
+
+
+## Supported Fields
+
+The following table lists the fields supported for filtering Flight Control objects:
+
+| Field Name                   | Field Type           |
+|------------------------------|----------------------|
+| `metadata.name`              | `String`             |
+| `metadata.owner`             | `String`             |
+| `metadata.labels`            | `StringArray`        |
+| `metadata.annotations`       | `StringArray`        |
+| `metadata.creationTimestamp` | `Timestamp`          |
+| `metadata.deletionTimestamp` | `Timestamp`          |
+| `spec`                       | `JSON`               |
+| `status`                     | `JSON`               |
+
+
+### Usage Examples
+
+Use the `--field-selector` flag to filter devices based on field values. Here are some examples:
+
+#### Example 1: Exclude a Specific Device by Name
+This command filters out a specific device by its name:
+```bash
+flightctl get devices --field-selector 'metadata.name!=c3tkb18x9fw32fzx5l556n0p0dracwbl4uiojxu19g2'
+```
+#### Example 2: Filter by Owner, Labels, and Creation Timestamp
+This command retrieves devices owned by `Fleet/pos-fleet`, located in the `us` region, and created in 2024:
+```bash
+flightctl get devices --field-selector 'metadata.owner=Fleet/pos-fleet, metadata.labels @> region\=us, metadata.creationTimestamp >= 2024-01-01T00:00:00Z, metadata.creationTimestamp < 2025-01-01T00:00:00Z'
+```
+#### Example 3: Filter by Owner, Labels, and Status
+This command retrieves devices owned by `Fleet/pos-fleet`, located in the `us` region, and with a `status.updated.status` of either "Unknown" or "OutOfDate":
+```bash
+flightctl get devices --field-selector 'metadata.owner=Fleet/pos-fleet, metadata.labels @> region\=us, status.updated.status in ("Unknown", "OutOfDate")'
+```
+
+### Fields Discovery
+
+Some Flight Control Objects might expose additional supported fields. You can discover the supported fields by using `flightctl` with the `--field-selector` option. If you attempt to use an unsupported field, the error message will list the available supported fields.
+
+For example:
+
+```bash
+flightctl get device --field-selector='text'
+
+Error: listing devices: 400, message: unknown or unsupported field: unable to resolve field name "text". Supported fields are: [metadata.annotations metadata.creationTimestamp metadata.deletionTimestamp spec metadata.owner metadata.labels metadata.alias status metadata.name metadata.nameoralias]
+```
+In this example, the field `text` is not a valid field for filtering. The error message provides a list of supported fields that can be used with `--field-selector` for the `device` object.
+
+You can then use one of the supported fields, as shown below:
+```bash
+flightctl get devices --field-selector 'metadata.alias @> cluster'
+```
+In this command, the `metadata.alias` field is checked with the containment operator `@>` to see if it contains the value `cluster`.
+
+
+### Fields Discovery
```

Comment:
"Fields Discovery" section is duplicate, with a few minor variations in each case.

Reply:
Fixed